### PR TITLE
Fix/condo/sberdoma 701/use employee user values

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
When user updates his name, email or phone, employees in his organization can view updated data

### CODE REVIEW

**Review by AntonAL — CHANGES_REQUESTED** [review]
Please, test updating of all employees of a user.

File: apps/condo/domains/organization/schema/OrganizationEmployee.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
 /**
  * Generated by `createschema organization.OrganizationEmployee 'organization:Relationship:Organization:CASCADE; user:Relationship:User:SET_NULL; inviteCode:Text; name:Text; email:Text; phone:Text; role:Relationship:OrganizationEmployeeRole:SET_NULL; isAccepted:Checkbox; isRejected:Checkbox' --force`
  */
+const { User } = require('../../user/utils/serverSchema')
```
Comment (toplenboren) [inline apps/condo/domains/organization/schema/OrganizationEmployee.js:4]: please use domain imports here

File: apps/condo/domains/user/schema/User.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
  * Generated by `createschema user.User name:Text; password?:Password; isAdmin?:Checkbox; email?:Text; isEmailVerified?:Checkbox; phone?:Text; isPhoneVerified?:Checkbox; avatar?:File; meta:Json; importId:Text;`
  */
 
+const { OrganizationEmployee } = require('../../organization/utils/serverSchema')
```
Comment (toplenboren) [inline apps/condo/domains/user/schema/User.js:5]: Please use domain imports here

File: apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.js [hunk 3]
```diff
@@ -16,12 +16,31 @@ const AcceptOrRejectOrganizationInviteService = new GQLCustomSchema('AcceptOrRej
...
                 if (!context.authedItem.id) throw new Error('[error] User is not authenticated')
                 const { id, data } = args
+                const authedItem = context.authedItem
                 let { isRejected, isAccepted, ...restData } = data
                 isRejected = isRejected || false
                 isAccepted = isAccepted || false
 
-                const obj = await updateOrganizationEmployee(context, id, { isRejected, isAccepted, ...restData })
-                return await getById('OrganizationEmployee', obj.id)
+                let employee
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.js:24]: it's better to move this logic inside Invite mutation.

Comment (Dimitreee) [reply apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.js:24]: +1

Comment (Dimitreee) [reply apps/condo/domains/organization/schema/AcceptOrRejectOrganizationInviteService.js:24]: @pahaz task need to be descussed

**Comment by nomerdvadcatpyat** [comment]
Decided that the data will be updated only for those employees who have confirmed the invitation

**Review by pahaz — APPROVED** [review]

**Review by Dimitreee — APPROVED** [review]

**Pull request closed without merging**